src/translations/en.json

```diff
@@ -67,5 +67,13 @@
     "playAiContent2":"to learn the ropes and test your mettle.",
     "gameName": "Game Name",
     "join":"Join"
+  },
+  "reauthentication": {
+    "reconnectToGame": "Reconnect to game",
+    "youHaveDisconnected": "You have disconnected due to inactivity. Log in again to resume your session",
+    "username": "Username",
+    "password": "Password",
+    "leaveGame": "Leave Game",
+    "login": "Log In"
```

Comment:
Since this dialog is only used inside the `GameView`, let's move this into the existing "game" block of translations. There are a number of dialogs in the game view so I'd like to group the text under "game.dialogs.reauthenticateDialog" so that each dialog gets its own chunk of text within the "game.dialogs" section. 

That way all the game text is grouped together, all the dialogs are together within that, and the text for each specific dialog is in a block named after the component using it.